index.js

```diff
@@ -0,0 +1,618 @@
+let size = 'small';
+
+function changeSizeforLarge(arr, array, j) {
+    let i = 0;
+    for (let elem of arr) {
+        if (i <= j) {
+
+            if (array[i][0] == 'ShiftRight' || array[i][0] == 'ShiftLeft') {
+                elem.innerHTML = 'Shift';
+            } else if (array[i][0] == 'ControlRight' || array[i][0] == 'ControlLeft' || array[i][1] == 'ControlLeft') {
+                elem.innerHTML = 'Control';
+            } else if (array[i][0] == 'AltRight' || array[i][0] == 'AltLeft') {
+                elem.innerHTML = 'Alt';
+            } else if (array[i][0] == 'Backquote') {
+                elem.innerHTML = '~';
+            } else if (array[i][0] == 'MetaLeft') {
+                elem.innerHTML = 'Win';
+            } else {
+                elem.innerHTML = array[i][1];
+            }
+        }
+        i++;
+    }
+    size = 'large';
+}
+
+function changeSizeforSmall(arr, array, j) {
+    let i = 0;
+    for (let elem of arr) {
+        if (i <= j) {
+            if (array[i][0] == 'ShiftRight' || array[i][0] == 'ShiftLeft') {
+                elem.innerHTML = 'Shift';
+            } else if (array[i][0] == 'ControlRight' || array[i][0] == 'ControlLeft' || array[i][1] == 'ControlLeft') {
+                elem.innerHTML = 'Control';
+            } else if (array[i][0] == 'AltRight' || array[i][0] == 'AltLeft') {
+                elem.innerHTML = 'Alt';
+            } else if (array[i][0] == 'Backquote') {
+                elem.innerHTML = '~';
+            } else if (array[i][0] == 'MetaLeft') {
+                elem.innerHTML = 'Win';
+            } else {
+                elem.innerHTML = array[i][0];
+            }
+        }
+
+        i++;
+    }
+    size = 'small';
+}
+
+
+///////////////////////////Created event click at the button
+keybourd.addEventListener('click', function(event) {
+    let button = event.target.closest('button');
+
+    if (!button) {
+        return
+    }
+
+    if (button.textContent != 'Backspace' && button.textContent != 'CapsLock' && button.textContent != 'Shift' &&
+        button.textContent != 'Tab' && button.textContent != 'Enter' && button.textContent != 'Space' &&
+        button.textContent != 'Alt' && button.textContent != 'Control' && button.textContent != 'Win') {
+        addAmimation(button);
+        textarea.value += button.textContent;
+    };
+
+    if (button.textContent == 'Space') {
+        addAmimation(button);
+        textarea.value += ' ';
+    }
+
+    if (button.textContent == 'Enter') {
+        addAmimation(button);
+        textarea.value += '\n';
+    }
+
+    if (button.textContent == 'Backspace') {
+        addAmimation(button);
+        textarea.value = textarea.value.slice(0, textarea.value.length - 1);
+    }
+
+
+    if (button.textContent == 'CapsLock') {
+        addAmimation(button);
+        if (size == 'small' && lg == 'english') {
+            changeSizeforLarge(document.querySelector('.row-2').children, englishKeybord[1], 10);
```

Comment:
It seams to me that I always see duplication of the code, it's normal for code jam, because you don't have enough time to create perfect architecture, but try to refactor such cases in the future.